package.json

```diff
@@ -38,11 +38,9 @@
     "execa": "^0.4.0",
     "karma": "^1.1.0",
     "karma-browserify": "^5.0.4",
-    "karma-chrome-launcher": "^1.0.1",
+    "karma-chrome-launcher": "^2.2.0",
```

Comment:
If you want, I can upgrade the other dependencies as well.

Reply:
👍

Reply:
Done